src/catalog/catalog.cpp

```diff
@@ -463,5 +463,11 @@ std::unordered_set<RelTableSchema*> Catalog::getAllRelTableSchemasContainBoundTa
     return relTableSchemas;
 }
 
+void Catalog::addVectorFunction(
```

Comment:
open an issue about renaming vector function to scalar function

Reply:
#1809